src/goto-programs/rw_set.h

```diff
@@ -87,7 +87,8 @@ class rw_sett
     bool w,
     const std::string &suffix,
     const guardt &guard,
-    const exprt &original_expr);
+    const exprt &original_expr,
+    bool dereferenced = false);
```

Comment:
Would it be possible to make `dereferenced` not a defaulted parameter but to pass it explicitly at every call-site? Or are there too many for this PR?

Reply:
Why do we need to explicitly call this parameter?